workflowitems.md

```diff
@@ -122,6 +122,21 @@ In addition, it allows in future to change the 1:1 association between collectio
 
 It returns the workflowitems created by the specified submitter
 
+## Get Single Workflow Item from Item UUID
+
+**/api/submission/workflowitems/item/<:uuid>**
```

Comment:
Please move that in the search methods area and name it
`/api/submission/workflowitems/search/item?uuid=<:item-uuid>`

Reply:
This again could alternatively be resolvable via a redirect. See this comment https://github.com/DSpace/Rest7Contract/pull/97#discussion_r376538335

Reply:
according to https://github.com/DSpace/Rest7Contract/pull/97#discussion_r377561234 as we already have search method that return single object I still recommend a search method here

Reply:
This would match with [WorkflowItemService.findByItem()](https://github.com/DSpace/DSpace/blob/a0ca6e2fadb60e718d60685ae4f9a7ed1efb2101/dspace-api/src/main/java/org/dspace/workflow/WorkflowItemService.java#L71), so it's a find method returning one workflow item.
I will adjust it as such in the contract